lib/ukreloc/Config.uk

```diff
@@ -0,0 +1,3 @@
+config LIBUKRELOC
+	bool "ukreloc: Self relocation and symbol based relocations"
+	default y
```

Comment:
`libukreloc` needs to be enabled by default to avoid surrounding calls to `ur_*` macros with `#ifdef's`. The `ur_*` macro's, when `CONFIG_OPTIMIZE_PIE` is disables, revert back to using absolute addressing.